src/main/kotlin/app/cash/licensee/pluginExtension.kt

```diff
@@ -213,6 +235,12 @@ enum class ViolationAction {
   IGNORE,
 }
 
+@Suppress("unused") // Public API.
+enum class UnusedAction {
+  WARN,
```

Comment:
I think should probably be named `LOG` to match the violation action.

Reply:
Good eye - updated